src/index.d.ts

```diff
@@ -1,91 +1,140 @@
+/*** This file was generated automatically. To recreate, please run `npm run build-index`. ***/
+
 import {
-    fetchSongById,
-    fetchArtists,
-    fetchSongArtistCount,
-    fetchRelatedSongs,
-    fetchAllSongs,
-    fetchSongFilterOptions,
-    fetchSongCount,
-    fetchWorkouts,
-    fetchNewReleases,
-    fetchLiveEvent,
-    fetchUpcomingEvents,
-    fetchByRailContentId,
-    fetchByRailContentIds,
-    fetchAll,
-    fetchAllFilterOptions,
-    fetchFoundation,
-    fetchMethods,
-    fetchMethod,
-    fetchMethodChildren,
-    fetchMethodNextLesson,
-    fetchMethodChildrenIds,
-    fetchNextPreviousLesson,
-    fetchRelatedLessons,
-    fetchAllPacks,
-    fetchPackAll,
-    fetchPackChildren,
-    fetchLessonContent,
-    fetchCourseOverview,
-    fetchChallengeOverview,
-    fetchScheduledReleases,
-    fetchArtistLessons,
-    fetchGenreLessons,
-  } from './services/sanity.js';
+	globalConfig,
+	initializeService
+} from './services/config.js';
 
-import { 
-    fetchCompletedState, 
-    fetchAllCompletedStates,
-    fetchContentInProgress,
-    fetchCompletedContent,
-    fetchVimeoData,
-    fetchContentPageUserData,
+import {
+	fetchAllCompletedStates,
+	fetchCompletedContent,
+	fetchCompletedState,
+	fetchContentInProgress,
+	fetchContentPageUserData,
+	fetchHandler,
+	fetchLikeContent,
+	fetchSongsInProgress,
+	fetchUnlikeContent,
+	fetchUserContext,
+	fetchVimeoData
 } from './services/railcontent.js';
```

Comment:
This adds entries that are exported in railcontent.js but were missing before:

- fetchHandler,
- fetchLikeContent,
- fetchSongsInProgress,
- fetchUnlikeContent,
- fetchUserContext,